afisha-server/server/api/errors/errorList.js

```diff
@@ -0,0 +1,6 @@
+const errorList = {
+    NOT_FOUND: 'The entity is not found',
```

Comment:
Please rework file structure. Let your error key contains object with error message (string) and internal error code (number)

And also - NOT_FOUND can be useful for any entity - users, events, comments. You have to create more speaking name. EVENT_NOT_FOUND suits more well.